src/gui/sdlmain.cpp

```diff
@@ -243,21 +256,20 @@ struct SDL_Block {
 			bool display_res;
 		} full;
 		struct {
-			Bit16u width, height;
+			uint16_t width = 0; // TODO convert to int
+			uint16_t height = 0; // TODO convert to int
+			bool use_original_size = true;
 		} window;
 		Bit8u bpp;
-		Bit32u sdl2pixelFormat;
 		bool fullscreen;
-		bool lazy_fullscreen;
-		bool lazy_fullscreen_req;
-		bool vsync;
+		bool vsync = false;
 		SCREEN_TYPES type;
 		SCREEN_TYPES want_type;
 	} desktop;
 #if C_OPENGL
 	struct {
 		SDL_GLContext context;
-		Bitu pitch;
+		int pitch = 0;
 		void * framebuf; // TODO Use unique_ptr to prevent leak
```

Comment:
`framebuf = nullptr`?

Reply:
Done :) Thanks for pointing this out. I also removed TODO comment, as this buffer should not be managed via unique_ptr after all.